cmake/snappy.cmake

```diff
@@ -20,8 +20,8 @@ include_guard()
 include(cmake/utils.cmake)
 
 FetchContent_DeclareGitHubWithMirror(snappy
-  google/snappy 1.1.7
-  MD5=40a371a653b7bb5a47df59b937b78557
+  google/snappy f725f6766bfc62418c6491b504c8e5865ec99412
```

Comment:
Didn't upstream release a tag for 1.10?

Reply:
No. Release tag age broken, so we need to use directly commit state

Reply:
Thanks for your explanation - I checked https://github.com/google/snappy/compare/1.1.10...main and understand what's tagged now.